modules/xmpp/JingleSessionPC.js

```diff
@@ -464,15 +465,15 @@ export default class JingleSessionPC extends JingleSession {
          * the value of RTCPeerConnection.iceConnectionState changes.
          */
         this.peerconnection.oniceconnectionstatechange = () => {
-            const now = window.performance.now();
             let isStable = false;
 
             if (!this.isP2P) {
-                this.room.connectionTimes[
-                    `ice.state.${this.peerconnection.iceConnectionState}`]
-                    = now;
+                window.performance.mark(`jitsi.ice.state.${this.peerconnection.iceConnectionState}`);
             }
-            logger.log(`(TIME) ICE ${this.peerconnection.iceConnectionState} ${this.isP2P ? 'P2P' : 'JVB'}:\t`, now);
+
+            logger.log(`ICE ${this.peerconnection.iceConnectionState} ${this.isP2P ? 'P2P' : 'JVB'}`);
```

Comment:
it will be still useful to have `now` printed. Especially since we still use it.

Reply:
As I mentioned in https://github.com/jitsi/lib-jitsi-meet/pull/1707#discussion_r706075330 that now() is not a timestamp. Also, the logger itself shows a timestamp, do we need another?